yellowbrick/features/rankd.py

```diff
@@ -126,18 +172,13 @@ class Rank2D(FeatureVisualizer):
     process, but can and should be set as early as possible.
     """
 
-    ranking_methods = {
```

Comment:
We should leave a `ranking_methods` class property on the base as just an empty dictionary I think; to ensure that a `KeyError` is raised by subclasses that don't implement this rather than an `AttributeError` -- I think that would help us more quickly debug.